src/extension/report.ts

```diff
@@ -0,0 +1,50 @@
+import * as Sentry from "@sentry/node";
+import { v4 as uuidv4 } from "uuid";
+import {
+    workspace,
+} from "vscode";
+
+export class CrashReporter {
```

Comment:
I would update the name of the file to be `crashreporter.ts` to make discoverability easier.